### PR TITLE
Add getSetoid for NonEmptyArray

### PR DESCRIPTION
**Pull request by MaximeRDY**
Hello, 

I've just add `getSetoid` on NonEmptyArray.

### CODE REVIEW

File: src/NonEmptyArray.ts [hunk 1]
```diff
@@ -523,6 +523,22 @@ export const getSemigroup = <A = never>(): Semigroup<NonEmptyArray<A>> => {
...
+ *
+ * @example
+ * import { NonEmptyArray, getSetoid } from 'fp-ts/lib/NonEmptyArray'
+ * import { setoidNumber } from 'fp-ts/lib/Setoid'
+ *
+ * const S = getSetoid(setoidNumber)
+ * assert.strictEqual(S.equals(new NonEmptyArray(1, []), new NonEmptyArray(1, [])), true)
+ * assert.strictEqual(S.equals(new NonEmptyArray(1, []), new NonEmptyArray(1, [2])), false)
+ *
+ * @since 1.13.0
```
Comment (gcanti) [inline src/NonEmptyArray.ts]: little typo, should be 1.14.0

File: src/NonEmptyArray.ts [hunk 2]
```diff
@@ -523,6 +523,22 @@ export const getSemigroup = <A = never>(): Semigroup<NonEmptyArray<A>> => {
...
+ */
+export const getSetoid = <A>(S: Setoid<A>): Setoid<NonEmptyArray<A>> => {
+  return contramap(x => x.toArray(), getArraySetoid(S))
```
Comment (gcanti) [inline src/NonEmptyArray.ts]: The body can be optimized

```ts
const setoidTail = getArraySetoid(S)
//     v-- checks for reference equality
return fromEquals((x, y) => S.equals(x.head, y.head) && setoidTail.equals(x.tail, y.tail))
//    bail out if heads are different --^
```

**Comment by MaximeRDY** [comment]
changes done ;-)

**Comment by gcanti** [comment]
Thanks @MaximeRDY

**Pull request merged**